python/lsst/daf/butler/registry/queries/expressions/parser/exprTree.py

```diff
@@ -240,11 +245,32 @@ def __str__(self) -> str:
         return "{value}".format(**vars(self))
 
 
+class UuidLiteral(LiteralNode):
+    """Node representing a UUID literal.
+
+    Parameters
+    ----------
+    value : `UUID`
+        Literal value.
+    """
+
+    def __init__(self, value: UUID):
+        Node.__init__(self)
+        self.value = value
+
+    def visit(self, visitor: TreeVisitor) -> Any:
+        # Docstring inherited from Node.visit
+        return visitor.visitUuidLiteral(self.value, self)
+
+    def __str__(self) -> str:
+        return "{value}".format(**vars(self))
```

Comment:
```suggestion
        return str(self.value)
```
seem like a much simpler way to write this.